### PR TITLE
katib python packages

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: py3-cloudpickle.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      README: 'CONFIRM WITH: curl -L https://files.pythonhosted.org/packages/5f/51/913ecca3970a2227cf4d5e8937df52cc28f465ac442216110b8e3323262d/cloudpickle-2.2.1.tar.gz | sha256sum'
```
Comment (rawlingsj) [inline py3-cloudpickle.yaml:26]: I think we can remove all the `README: 'CONFIRM WITH: curl -L ...` lines in this PR, they're not needed

Comment (rawlingsj) [reply py3-cloudpickle.yaml:26]: Other than that I think it looks good!

File: py3-cloudpickle.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Generated from https://pypi.org/project/cloudpickle/
```
Comment (rawlingsj) [inline py3-cloudpickle.yaml:1]: not sure if we need these at the top of each either, not a blocker but feels like we we're tidying up our configs and making them more consistent.  So maybe drop all the `# Generated from...`

File: py3-cloudpickle.yaml (hunk 1 above)
Comment (developer-guy) [reply py3-cloudpickle.yaml]: done! thank you!

File: py3-cloudpickle.yaml (hunk 2 above)
Comment (developer-guy) [reply py3-cloudpickle.yaml]: removed!

**Review by rawlingsj — CHANGES_REQUESTED** [review]
similar to https://github.com/wolfi-dev/os/pull/4558#issuecomment-1684480938 I just noticed that there's sha's in the fetch URL.

I'm sorry for the extra work but I think these need to be git-checkouts and have the update config track github rather than release monitor

**Comment by developer-guy** [comment]
> similar to [#4558 (comment)](https://github.com/wolfi-dev/os/pull/4558#issuecomment-1684480938) I just noticed that there's sha's in the fetch URL.
> 
> I'm sorry for the extra work but I think these need to be git-checkouts and have the update config track github rather than release monitor

Yes, I tried using `git-checkout` for those who work with `git-checkout` but some of them didn't work, so I kept using fetch from `files.pythonhosted.org` since APKBUILD does the same thing.

**Comment by rawlingsj** [comment]
> Yes, I tried using git-checkout for those who work with git-checkout but some of them didn't work, so I kept using fetch from files.pythonhosted.org since APKBUILD does the same thing.

do you have more details of what wasn't working?  I just tried here switching the first two packages in this PR to github and the builds worked.  I.e.

```yaml
package:
  name: py3-cloudpickle
  version: 2.2.1
  epoch: 0
  description: Extended pickling support for Python objects
  copyright:
    - license: BSD 3-Clause License
  dependencies:
    runtime:
      - python3

environment:
  contents:
    packages:
      - ca-certificates-bundle
      - wolfi-base
      - busybox
      - build-base
      - python3
      - py3-setuptools

pipeline:
  - uses: git-checkout
    with:
      repository: https://github.com/cloudpipe/cloudpickle
      expected-commit: 917bed6bc63355f50aeb4f001683be7f172b6b88
      tag: v${{package.version}}

  - name: Python Build
    runs: python setup.py build

  - name: Python Install
    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"

  - uses: strip

update:
  ignore-regex-patterns:
    - ^\.
  enabled: true
  github:
    identifier: cloudpipe/cloudpickle
    strip-prefix: v
    use-tag: true
    tag-filter: v
```

and 

```yaml
package:
  name: py3-future
  version: 0.18.3
  epoch: 0
  description: Clean single-source support for Python 3 and 2
  copyright:
    - license: MIT
  dependencies:
    runtime:
      - python3

environment:
  contents:
    packages:
      - ca-certificates-bundle
      - wolfi-base
      - busybox
      - build-base
      - python3
      - py3-setuptools

pipeline:
  - uses: git-checkout
    with:
      repository: https://github.com/PythonCharmers/python-future
      expected-commit: af1db970b0879b59e7aeb798c27a623144561cff
      tag: v${{package.version}}

  - name: Python Build
    runs: python setup.py build

  - name: Python Install
    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"

  - uses: strip

update:
  enabled: true
  github:
    identifier: PythonCharmers/python-future
    strip-prefix: v
```

> since APKBUILD does the same thing.

The problem here is that we use different tooling to alpine, using APKBUILDs as a reference can help but we don't always want to do the exact same thing as wolfi is different.

File: py3-snowballstemmer.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 09b16deb8547d3412ad7b590689584cd0fe25ec8db3be37788be3810cbf19cb1
+      uri: https://files.pythonhosted.org/packages/44/7b/af302bebf22c749c56c9c3e8ae13190b5b5db37a33d9068652e8f73b7089/snowballstemmer-${{package.version}}.tar.gz
```
Comment (rawlingsj) [inline py3-snowballstemmer.yaml:26]: can this also be a `git-checkout` https://github.com/snowballstem/snowball/releases/tag/v2.2.0?

Comment (developer-guy) [reply py3-snowballstemmer.yaml:26]: I tried but the "setup.py" does not exist on GitHub source but it exists on files.pythonhosted source, so, I couldn't understand how to build with by using GitHub source

Comment (kaniini) [reply py3-snowballstemmer.yaml:26]: Lets stick with the tarball for now.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**